xbmc/cores/VideoPlayer/VideoRenderers/CMakeLists.txt

```diff
@@ -39,10 +39,7 @@ if(OPENGL_FOUND)
                       FrameBufferObject.h)
 endif()
 
-if(OPENGLES_FOUND AND (NOT CORE_PLATFORM_NAME STREQUAL rbpi AND
-                       NOT CORE_PLATFORM_NAME STREQUAL aml AND
-                       NOT CORE_PLATFORM_NAME STREQUAL android AND
-                       NOT CORE_PLATFORM_NAME STREQUAL imx))
+if(OPENGLES_FOUND AND NOT CORE_PLATFORM_NAME STREQUAL rbpi)
```

Comment:
actually this does not depend on whether GLES was found or not. it depends on if the platform depends of GLES renderer or not. means the logic here is not correct.

Reply:
Ok I revised it to just depend on which platforms need it